### PR TITLE
refactor: refactor setup template for vue3

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [x] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue [hunk 1]
```diff
@@ -1,3 +1,141 @@
...
+    (e: 'update:visible-menu', value?: boolean): void;
+    (e: 'focus-menu'): void;
+    (e: 'click-delete'): void;
+}>();
+
+const slots = useSlots();
+
+const state = reactive({
+    proxyVisibleMenu: useProxyValue<boolean | undefined>('visibleMenu', props, emit),
+    contextMenuRef: null as null|any,
```
Comment (piggggggggy) [inline packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:75]: nitpick: Maybe this Ref would be not working..
use `toRef` or change to single `Ref` like `containerRef`.

**Review by piggggggggy — APPROVED** [review]
LGTM!~!!~!
I left a comment.

File: packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue (hunk 1 above)
Comment (skdud4659) [reply packages/mirinae/src/inputs/dropdown/select-dropdown/PSelectDropdown.vue:75]: fixed it!

**Review by yuda110 — APPROVED** [review]
LGTM!!!!!!!

**Pull request merged**